public/css/app.css

```diff
@@ -9,4 +9,14 @@ p{
     line-height: 1.6;
     text-align: justify;
 }
+a{
+    text-decoration: none;
+}
+
+/* this means that all articles except the first one*/
```

Comment:
misleading comment. this css is applies to all articles immediately after article